api/bus.go

```diff
@@ -722,3 +722,8 @@ type AddPartialSlabResponse struct {
 func FormatETag(ETag string) string {
```

Comment:
Leaving a comment to prevent this from getting merged by accident. After approval I'd like to test it on a node before merging.